README.md

```diff
@@ -30,3 +34,15 @@ Be sure to read [How To submit your suggestion](./how-to/submit-your-suggestion.
 - [@marwane.dev](https://www.instagram.com/marwane.dev/)
 - [@programmer.nest](https://www.instagram.com/programmer.nest/)
 - [@webapp_creator](https://www.instagram.com/webapp_creator/)
+
+## Linkedin profiles 
+-[Christine Belzie](https://www.linkedin.com/in/christinebelzie)
```

Comment:
Is the idea of Linkedin profiles to add in our community members?
Or more of a place to add groups on linked in that align with SheSharp?

Reply:
Good question @TessavWalstijn! It's the former.